components/mediation/inbound-endpoints/org.wso2.micro.integrator.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/inboundfactory/InboundRequestProcessorFactoryImpl.java

```diff
@@ -44,6 +44,9 @@
 
     public static enum Protocols {jms, file, http, https, hl7, kafka, mqtt, rabbitmq, ws, wss, grpc}
 
+    //System property to enable/disable portOffset for Inbound Endpoints
+    private static boolean enableInboundPortOffset = Boolean.getBoolean("inboundPortOffset");
```

Comment:
Let's make the name self explanatory, something like disableInboundPortOffset/enableInboundPortOffset ?

Reply:
Also, do we need to add the property to the scripts?

Reply:
Updated the scripts and renamed the system property name to 'enableInboundPortOffset'